test/common/http/codec_impl_corpus/clusterfuzz-testcase-minimized-codec_impl_fuzz_test-5074389318107136-h1-with-trailers

```diff
@@ -0,0 +1,232 @@
+h1_settings {
+  server {
```

Comment:
nit: Let's add a client entry here, with `enable_trailers` set to false.

Reply:
`enable_trailers: false` for client does not resolve the issue. Tailers need to be allowed for both client and server to make the testcase pass. I have fixed that now in the corpus and in the test, see latest commit https://github.com/envoyproxy/envoy/pull/21124/commits/cac67cb8ef41bd055ff12af0cb2970e96f57f6d4

Reply:
I'm assuming that the fuzzer will randomly assign `enable_trailers: false` somewhere down the road, so wanted to make sure that this works. Setting it to true, as you did, also works.